translations/README-ja.md

```diff
@@ -0,0 +1,98 @@
+<img alt="LitmusChaos" src="https://landscape.cncf.io/logos/litmus.svg" width="200" align="left">
+
+# Litmus
+### クラウドネイティブ・カオスエンジニアリング
```

Comment:
We'd love to welcome this in our README, can you please send a separate PR only for this!

Reply:
@S-ayanide Don't know how this happened but none of changes were made by me in my fork repo. I'll add this README in Japanese and German and send different PR.